packages/drizzle/src/find/traverseFields.ts

```diff
@@ -91,8 +91,10 @@ export const traverseFields = ({
             with: {},
           }
 
+          const sanitizedPathForTableName = path ? `${toSnakeCase(path.slice(0, -1))}_` : ''
```

Comment:
This appears to be redundant: `${toSnakeCase(path.slice(0, -1))}_`
Shouldn't it just be `toSnakeCase(path)`?

Reply:
This is because of `toSnakeCase` works, input `camelCaseGroup_`:
![image](https://github.com/user-attachments/assets/addfa1bb-a8bc-4423-ac2d-9ea9a3756789)
But for `camelCaseGroup`:
![image](https://github.com/user-attachments/assets/265d35f3-c124-4b5a-8e69-38f8922c3e96)
`_` on the end because I added it. ``${toSnakeCase(path.slice(0, -1))}_``

Reply:
I wonder about a nested group into a group case, I think I want to write a test for it too

Reply:
As I thought, it still was failing for nested camelCase group / tabs
For example this (from my new test case)

```ts
{
  name: 'camelCaseGroup',
  type: 'group',
  fields: [
    {
      name: 'nesGroup',
      type: 'group',
      fields: [
        {
          type: 'array',
          name: 'arr',
          fields: [
            {
              type: 'text',
              name: 'text',
              localized: true,
            },
          ],
        },
      ],
    },
  ],
},
```

 Now I ensured that it works here as well and a bit changed the way how I sanitize the path specifically for table names. I don't think it's doable without introducing an additional argument into `traverseFields`